src/admin_extra/service_admin_extra_gdpr.erl

```diff
@@ -57,7 +57,10 @@ retrieve_all(Username, Domain, ResultFilePath) ->
 -spec modules_with_personal_data() -> [module()].
 modules_with_personal_data() ->
     [
-     mod_vcard
+     mod_vcard,
+     mod_offline_mnesia,
```

Comment:
Should call `mod_offline`, not the backends directly.

Reply:
yeah makes sense